src/lib/uTop.ml

```diff
@@ -35,8 +40,8 @@ let version = "%%VERSION%%"
    +-----------------------------------------------------------------+ *)
 
 let history = LTerm_history.create []
-let unopt = function
-        | None -> "None"
+let rec unopt = function
+        | None -> unopt(H.home_dir)
```

Comment:
if `H.home_dir` is `None`, this will loop indefinitely...

Reply:
It's never going to happen that H.home_dir is none.

Reply:
But I'll change it, sure.